eclipse-theia/theia-cpp-extensions
# enable linting in project

Signed-off-by: Marc Dumais <marc.dumais@ericsson.com>

#### What it does
This PR adds required configs so that linting (tslint) will work, as part of the TypeScript language server and report problems as-one-develops.

#### How to test
Open `.ts` files and intentionally break configured linting rules (e.g. 2 consecutive blank lines, remote semicolon at end of line). Verify that the linter report errors and that visual feedback is provided in the Theia editor.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
